Language_Parser/actionVerbs.py

```diff
@@ -733,36 +775,26 @@ def use(info):
     currentRoom = player.getLocation()
     allItems = player.getInventory() + currentRoom.getAccessibleItems()
 
-    # Identify the number of items provided
-    try:
+    # Handles "use" on one item (if item2 doesn't exist)
+    if len(info["Items"]) == 1:
+        item = info["Items"][0]
+        item = polaroidHelper(player, item, allItems)
+        response = useSingleItem(player, currentRoom, item)
+        if response == "":
+            return
```

Comment:
If the player tries to use a single item that doesn't have a response will this print the error message? Or just output an empty line?

Reply:
It only returns the "" when the verbHelper succeeds in printing a response on its own, so there is still a response given to the player either way.

Reply:
Ahh yes, I see it now. Thanks for explaining that for me!